airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1490,6 +1490,142 @@ paths:
+      tags: [Authentication]
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuthInfo'
+
+  /auth/login:
+    post:
+      summary: Login user
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: auth_login
+      tags: [Authentication]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AuthDBLoginForm'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JwtAuthAndUser'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
+
+
+  /auth-oauth/{provider}:
+    get:
+      summary: OAUTH user login
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: auth_oauthlogin
+      tags: [Authentication]
+      parameters:
+        - $ref: '#/components/parameters/ShouldRegister'
+        - $ref: '#/components/parameters/OauthProvider'
+        - $ref: '#/components/parameters/RedirectUrl'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuthorizationUrl'
+
+  /oauth-authorized/{provider}:
+    parameters:
+      - $ref: '#/components/parameters/OauthProvider'
+    get:
+      summary: Oauth authentication
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: authorize_oauth
+      tags: [Authentication]
+      parameters:
+        - $ref: '#/components/parameters/AuthState'
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JwtAuthAndUser'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
+
+
+  /auth-remoteuser:
+    get:
+      summary: Authenticate using remote user auth
+      x-openapi-router-controller: airflow.api_connexion.endpoints.auth_endpoint
+      operationId: auth_remoteuser
+      tags: [Authentication]
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JwtAuthAndUser'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
+
+  /refresh:
+    post:
+      summary: Refresh a token
+      description: Accepts fresh token and creates access token
```

Comment:
Typo: it should say `Accepts refresh token` not `Accepts fresh token`

Reply:
Good catch!